### DISCUSSION TITLE
Different CPU temp readings between GUI and DLL

### DISCUSSION THREAD

**Discussion by Ousirie** [LibreHardwareMonitor/LibreHardwareMonitor#1014] [General]
Testing on an HP Probook 450 G4 with an Intel i5-7200U (W10). GUI temp reading for CPU package sensor is always around 10c lower than what a PS script pulls from the DLL. Double checked that I'm monitoring the same Temperature sensor (and not distance to Tjmax) and tested both on idle and during load. HWinfo seems to agree with LHM's GUI temps.  Why would these readings differ? Does raw CPU package temp data need some sort of correction before being displayed?

**Comment by welpdx** [comment]
I have the same issue #1186
